apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -29,52 +30,56 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     marginVertical: 5,
   },
-  pickerContainer: {
-    flexDirection: 'row',
-    justifyContent: 'space-evenly',
-  },
   stackStyle: {
     borderWidth: 2,
     padding: 12,
     margin: 8,
   },
 });
 
-const getSwatchColorStyle = (name: string, color: ColorValue): ViewStyle => {
-  styles[name] = styles[name] || { backgroundColor: color };
-  return styles[name];
+const getSwatchColorStyle = (colorName: string, colorValue: ColorValue): ViewStyle => {
+  styles[colorName] = styles[colorName] || { backgroundColor: colorValue };
+  return styles[colorName];
 };
 
-type ColorTokenProps = { color: ColorValue; name: string };
+type ColorTokenProps = { colorValue: ColorValue; colorName: string };
 const ColorToken: React.FunctionComponent<ColorTokenProps> = (p: ColorTokenProps) => {
+  if (p.colorValue === undefined) {
+    console.warn('Color token ' + p.colorName + ' is undefined');
+  }
+
   const themedStyles = getThemedStyles(useTheme());
   return (
     <View style={styles.swatchItem}>
       <View
-        style={[getSwatchColorStyle(p.name, p.color), themedStyles.swatch]}
+        style={[getSwatchColorStyle(p.colorName, p.colorValue), themedStyles.swatch]}
         /* For Android E2E testing purposes, testProps must be passed in after accessibilityLabel. */
         {...testProps(COLORTOKENS_TEST_COMPONENT)}
       />
-      <Text>{p.name}</Text>
+      <View>
```

Comment:
nit: is there a particular reason for this View?

Reply:
Oh thanks for catching that, no I accidentally left that in when I was trying to figure out a way to get the text to wrap nicely on iOS, will remove